deploy/kubernetes/helm/che/templates/_keycloakHostHelper.tpl

```diff
@@ -1,8 +1,6 @@
 {{- define "keycloakHost" }}
-{{- if eq .Values.global.serverStrategy "default-host" }}
+{{- if or (eq .Values.global.serverStrategy "default-host") (eq .Values.global.serverStrategy "single-host") }}
```

Comment:
@i300543 Can you elaborate on why removing `che-<namepsace>` prefix from `keycloakHost` and `cheHost` would not change the behavior?

Reply:
The behavior would change:
lets say our ingressDomain is my-dev.myhost.com, and the namespace is my-dev
with the old logic the che wsmaster host then would be http(s)://che-my-dev.my-dev.myhost.com
and keycloak would be :  http(s)://che-my-dev.my-dev.myhost.com/auth
with the new logic the host would be: 
http(s)://my-dev.myhost.com (for wsmaster)
http(s)://my-dev.myhost.com/auth (for keycloak)
there is no real reason (as far as i understand) to include the namespace into the host name in case of single-host server strategy

Reply:
regarding the cheHost change - (in kc_realm_user.sh)
this affects the CORS headers  (allowed origins) and allowed redirect hosts, and this value should be the same as the che host, which is calculated by the _keycloakHostHelper.tpl, and not hard coded as the same logic which runs in one of the if's in the tpl script